app/modules/ads/tracking/slot-tracker.js

```diff
@@ -10,6 +10,7 @@ export const onChangeStatusToTrack = [
   'catlapsed',
   'error',
   'hivi-collapse',
+  'overscrolled',
```

Comment:
Seems not needed. `emitEvent` on AdSlot doesn't trigger statusChange, but customEvent (which is always tracked).

Reply:
👍